### PR TITLE
Log account id and all bidder names when recovering from OpenRTB auction bidder…

### PR DESCRIPTION
**Pull request by bsardo**
… panic (#446)

### CODE REVIEW

File: exchange/exchange.go [hunk 1]
```diff
@@ -369,11 +369,25 @@ func (e *exchange) getAllBids(ctx context.Context, cleanRequests map[openrtb_ext
...
 }
 
-func (e *exchange) recoverSafely(inner func(openrtb_ext.BidderName, openrtb_ext.BidderName, *openrtb.BidRequest, *pbsmetrics.AdapterLabels, currencies.Conversions), chBids chan *bidResponseWrapper) func(openrtb_ext.BidderName, openrtb_ext.BidderName, *openrtb.BidRequest, *pbsmetrics.AdapterLabels, currencies.Conversions) {
+func (e *exchange) recoverSafely(cleanRequests map[openrtb_ext.BidderName]*openrtb.BidRequest, inner func(openrtb_ext.BidderName, openrtb_ext.BidderName, *openrtb.BidRequest, *pbsmetrics.AdapterLabels, currencies.Conversions), chBids chan *bidResponseWrapper) func(openrtb_ext.BidderName, openrtb_ext.BidderName, *openrtb.BidRequest, *pbsmetrics.AdapterLabels, currencies.Conversions) {
 	return func(aName openrtb_ext.BidderName, coreBidder openrtb_ext.BidderName, request *openrtb.BidRequest, bidlabels *pbsmetrics.AdapterLabels, conversions currencies.Conversions) {
 		defer func() {
 			if r := recover(); r != nil {
-				glog.Errorf("OpenRTB auction recovered panic from Bidder %s: %v. Stack trace is: %v", coreBidder, r, string(debug.Stack()))
+
+				var allBidders string = ""
```
Comment (SyntaxNode) [inline exchange/exchange.go:377]: Syntax Nitpick: This is more commonly written as `allBidders := ""`

File: exchange/exchange.go [hunk 2]
```diff
@@ -369,11 +369,25 @@ func (e *exchange) getAllBids(ctx context.Context, cleanRequests map[openrtb_ext
...
+				var sb strings.Builder
```
Comment (SyntaxNode) [inline exchange/exchange.go:378]: Syntax Nitpick: I've seen a lot of code written this way, so it's cool with me. If you're interested, the non-var alternative is: `sb  := strings.Builder{}`

File: exchange/exchange.go [hunk 3]
```diff
@@ -369,11 +369,25 @@ func (e *exchange) getAllBids(ctx context.Context, cleanRequests map[openrtb_ext
...
+				for k := range cleanRequests {
+					sb.WriteString(string(k))
+					sb.WriteString(",")
+				}
+				if sb.Len() > 0 {
+					allBidders = sb.String()[:sb.Len()-1]
+				}
+
+				glog.Errorf(
```
Comment (SyntaxNode) [inline exchange/exchange.go:387]: Syntax Nitpick: I would recommend the first line of the string to be moved up a line.. something like:
```
glog.Errorf("OpenRTB auction recovered panic from Bidder %s: %v. "+
    "Account id: %s, All Bidders: %s, Stack trace is: %v",
coreBidder, r, bidlabels.PubID, allBidders, string(debug.Stack()))
```

File: exchange/exchange.go (hunk 1 above)
Comment (bsardo) [reply exchange/exchange.go]: 👍

**Pull request merged**